clang/test/CodeGenCoroutines/coro-elide-thinlto.cpp

```diff
@@ -0,0 +1,77 @@
+// This tests that the coroutine elide optimization could happen succesfully with ThinLTO.
+// This test is adapted from coro-elide.cpp and splits functions into two files.
+//
+// RUN: split-file %s %t
+// RUN: %clang --target=x86_64-linux -std=c++20 -O2 -flto=thin -I %S -c %t/coro-elide-callee.cpp -o coro-elide-callee.o
+// RUN: %clang --target=x86_64-linux -std=c++20 -O2 -flto=thin -I %S -c %t/coro-elide-caller.cpp -o coro-elide-caller.o
```

Comment:
This test case requires an X86 backend. Those of us who have embraced the AArch64-only future are encountering test errors when running this test.

(maybe it's just me)

Reply:
Just landed #90672 to limit the test to x86 linux targets. Did that fix the test failure?

Reply:
Yep! Thank you for the quick fix.